apps/sms/views/shared/js/utils.js

```diff
@@ -942,6 +942,21 @@
       return promise.then(
         () => shimHostIframe.contentWindow.bootstrap(appInstanceId)
       );
+    },
+
+    onceDocumentIsVisible() {
```

Comment:
Maybe the name is not the best, but I feel like we need something like this for a long time already :) What do you think?